proto/options.proto

```diff
@@ -43,6 +43,21 @@ enum ShapeFormat {
   geojson = 2;
 }
 
+// use this to select which top level fields should be present in the pbf output
+// the actions below are marked for each field that would provide a minimal response
+message PbfFieldSelector {
+  bool options = 1;
+  bool trip = 2;       // /trace_attributes
+  bool directions = 3; // /route /trace_route /optimized_route /centroid
+  bool status = 4;     // /status
```

Comment:
these are the parts of the response you can turn on and off. by default they are all on (ie if you omit setting this option at all in the larger `options` object.